### PR TITLE
Remove zingo lib dep

### PR DESCRIPTION
**Pull request by zancas**

### CODE REVIEW

File: tests/integration.rs [hunk 1]
```diff
@@ -1,16 +1,8 @@
...
-
 use std::path::PathBuf;
 
-use zcash_protocol::{PoolType, ShieldedProtocol};
-
-use zingolib::{
-    testutils::lightclient::{from_inputs, get_base_address},
-    testvectors::REG_O_ADDR_FROM_ABANDONART,
-};
+use testvectors::REG_O_ADDR_FROM_ABANDONART;
```
Comment (ala-mode) [inline tests/integration.rs:3]: I see `testvectors` here, seems to be depended on with line 15 of `Cargo.toml`

**Review by ala-mode — COMMENTED** [review]
running `git fetch --all --tags` results in:
```
Fetching zingo
remote: Enumerating objects: 179, done.
remote: Counting objects: 100% (176/176), done.
remote: Compressing objects: 100% (68/68), done.
remote: Total 179 (delta 121), reused 154 (delta 102), pack-reused 3 (from 2)
Receiving objects: 100% (179/179), 74.13 KiB | 145.00 KiB/s, done.
Resolving deltas: 100% (121/121), completed with 58 local objects.
From https://github.com/zingolabs/zingolib
   f9f3e5f6..56383d50  dev               -> zingo/dev
 ! [rejected]          mob-release-1.3.5 -> mob-release-1.3.5  (would clobber existing tag)
error: Could not fetch zingo
...

I'll look into overcoming this, otherwise My only real questions were about depending on `zingolib` in the `Cargo.toml` for this branch that is named `Remove zingo lib dep`

```

**Comment by ala-mode** [comment]
Running `git fetch --all --tags --force` worked for me to sort out my local repo.

File: Cargo.toml [hunk 2]
```diff
@@ -3,27 +3,16 @@ name = "zcash_local_net"
...
-# Lightclient
-zingo-netutils = { git = "https://github.com/zingolabs/zingolib.git", tag = "zaino_dep_004", optional = true }
-zingolib = { git = "https://github.com/zingolabs/zingolib.git", tag = "zaino_dep_004", features = [ "test-elevation" ], optional = true }
-
-# RPC
-tonic = {version = "0.12.2", features = ["tls", "tls-roots", "tls-webpki-roots"], optional = true }
+zebra-node-services = { git = "https://github.com/ZcashFoundation/zebra.git", tag = "v2.1.0", features = [ "rpc-client" ] }
+zebra-rpc = { git = "https://github.com/ZcashFoundation/zebra.git", tag = "v2.1.0", features = [ "getblocktemplate-rpcs" ] }
+zebra-chain = { git = "https://github.com/ZcashFoundation/zebra.git", tag = "v2.1.0" }
+testvectors = { git = "https://github.com/zancas/zingolib.git", branch = "publish_testvectors_as_crate" }
```
[the remark this answers is not in the document]
Comment (zancas) [reply Cargo.toml:15]: Right..  it depends on the `zingolib` git repository, which is the all the source code for several Rust crates collected into a "workspace".

What it's **independent** of is the `zingolib` library crate.

**Comment by zancas** [comment]
This will be superceded.

**Pull request closed without merging**